lib/inspec/resources/opa_cli.rb

```diff
@@ -0,0 +1,42 @@
+require "inspec/resources/opa"
+
+module Inspec::Resources
+  class OpaCli < Opa
+    name "opa_cli"
+    supports platform: "unix"
+    supports platform: "windows"
+
+    attr_reader :allow
+
+    def initialize(opts = {})
+      @policy = opts[:policy] || nil
+      @data = opts[:data] || nil
+      @query = opts[:query] || nil
+      fail_resource "policy and data are the mandatory for executing OPA." if @policy.nil? && @data.nil?
+      @content = load_result
+      super(@content)
+    end
+
+    def allow
+      @content["result"][0]["expressions"][0]["value"] if @content["result"][0]["expressions"][0]["text"].include?("allow")
+    end
+
+    def to_s
+      "OPA cli"
+    end
+
+    private
+
+    def load_result
+      raise Inspec::Exceptions::ResourceFailed, "#{resource_exception_message}" if resource_failed?
+
+      result = inspec.command("opa eval -i '#{@data}' -d '#{@policy}' '#{@query}'")
```

Comment:
You will probably want to make the path to the opa command an option.

Reply:
I was assuming that the user will export that in the PATH variable but this can be good option too.